components/Tabs/Tabs.js

```diff
@@ -2,61 +2,61 @@
 class TabLink {
   constructor(element) {
     // Assign this.element to the passed in DOM element
-    // this.element;
+    this.element = element;
     
     // Get the custom data attribute on the Link
-    // this.data;
+    this.data = document.querySelector(`.tabs-link[data-tab='${element.dataset.tab}']`);
```

Comment:
be careful about what's being asked here and what you're actually doing -- you only needed to grab the data-tab values on the element being passed in, which should have been read with `element.dataset.tab`.

instead, you're grabbing the first `element` with the `tabs-link` class and has the same data-tab value as the element being passed in... so essentially you're grabbing the `element` already being passed in a second time